src/viperleed/gui/measure/camera/badpixels.py

```diff
@@ -1415,3 +1406,33 @@ def __has_info(self):
     def __bool__(self):
         """Return the truth value of self."""
         return bool(self.__has_info and self.n_bad_pixels_sensor)
+
+
+class BadPixelsFrameStorage:
+    """Class for containing bad pixel frames."""
+
+    def __init__(self, height, width):
+        """Initialize bad pixel frame storage class."""
+        self._count = 0
+        self._sum = np.zeros((height, width), dtype=np.uint64)
+        self._sum_squares = np.zeros((height, width), dtype=np.uint64)
+
+    @property
+    def frame_sum(self):
+        """Return the summed up frames."""
+        return self._sum
+
+    def add_frame(self, frame):
+        """Add frame to frame storage."""
+        frame = frame.astype(np.uint64, copy=False)
+        self._count += 1
+        self._sum += frame
+        self._sum_squares += frame**2
+
+    def mean(self):
+        """Return the mean of the stored frames."""
+        return self._sum / self._count
+
+    def var(self):
+        """Return the variance calculated from the stored frames."""
+        return (self._sum_squares - self._sum**2 / self._count) / self._count
```

Comment:
```suggestion
        mean_of_squares = self._sum_squares / self._count
        return mean_of_squares - self.mean()**2
```

For reading it more easily (unless there may be some uncaught overflow errors).